apps/shipments/filters.py

```diff
@@ -30,7 +32,7 @@ class Meta:
                   'final_destination_location__name', 'final_destination_location__city', 'ship_from_location',
                   'final_destination_location__state', 'final_destination_location__address_1', 'ship_to_location',
                   'final_destination_location__postal_code', 'final_destination_location__country',
-                  'final_destination_location', 'ship_from_location__state', 'state', 'exception', 'delayed',
+                  'final_destination_location', 'ship_from_location__state', 'state', 'delayed',
```

Comment:
Should the exception field be removed here?

Reply:
Yes, it is removed from there and into it's own specific CharField filter with the method, so that you can query against the name of it instead of the integer value

Reply:
I'm not as familiar with FilterSets. Does this mean that `state` should also be removed here?

Reply:
hmm, it porbably should actually, since we are defining that filter too.